src/config.cpp

```diff
@@ -26,6 +26,9 @@
 #include "fortune.hpp"
 #include "server_lock.hpp"
 
+#include <boost/filesystem.hpp>
+namespace fs = boost::filesystem;
```

Comment:
We should use `std::filesystem` instead of `boost::filesystem`. This project is compiled with C++17 support, and I am using GCC 9.3.0 as the minimum version for my testing.

Reply:
sure, thanks, wasn't aware of that :)